alphabot/bot.py

```diff
@@ -90,15 +93,16 @@ def _setup_memory(self, memory_type='dict'):
         # TODO: memory module should provide this mapping.
         memory_map = {
             'dict': memory.MemoryDict,
+            'redis': memory.MemoryRedis,
         }
 
-        # Get assiciated memory class or default to Dict memory type.
-        NewMemory = memory_map.get(memory_type)
-        if not NewMemory:
+        # Get associated memory class or default to Dict memory type.
+        memory_type = memory_map.get(memory_type)
```

Comment:
I actually prefer keeping this variable named same style a class name would be. This makes it a bit more apparent later on when you say `obj = NewMemory()` rather than `variable = memory_type()`. The latter makes it syntactically look like a function call.